ocs_ci/ocs/resources/pvc.py

```diff
@@ -245,6 +245,20 @@ def create_snapshot(self, snapshot_name=None, wait=False):
         snapshot_obj.parent_volume_mode = self.get()["spec"]["volumeMode"]
         return snapshot_obj
 
+    def create_reclaim_space_job(self):
+        """
+        Create ReclaimSpaceJob to invoke reclaim space operation on RBD volume
+
+        Returns:
+            OCS object
+
+        """
+        assert (
+            self.provisioner == "openshift-storage.rbd.csi.ceph.com"
```

Comment:
please save "openshift-storage.rbd.csi.ceph.com" in constants.py

Reply:
Added RBD_PROVISIONER